.gitignore

```diff
@@ -11,6 +11,7 @@ tmp/
 zf-mkdoc-theme/
 
 clover.xml
+composer.lock
```

Comment:
This should be removed; the travis setup uses the lowest/locked/latest strategy, so the file needs to be present. I'll re-add it on merge.